modules/virt-adding-a-boot-source-web.adoc

```diff
@@ -63,4 +52,4 @@ Provided boot sources are updated automatically to the latest version of the ope
 
 .. Click *Save and clone* if you cloned an existing PVC.
 
-Your custom virtual machine template with a boot source is listed in the *Templates* page, and you can create virtual machines by using this template.
+Your custom virtual machine template with a boot source is listed on the *Catalog* page. You can use this template to quick create a virtual machine.
```

Comment:
"quick create" sounds odd to me. I think I've seen that was the terminology used in the UI, but just wondering if that's appropriate to use here, or if we should say "quickly create" instead